File: Course_01/C Language/EstruturaRepetitiva/combustivel.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int cod, count1, count2, count3;

    cod = 0;
    count1 = 0;
    count2 = 0;
    count3 = 0;

    while (cod != 4) {
        printf("Informe um codigo (1, 2, 3) ou 4 para parar: ");
        scanf("%d", &cod);

        if (cod == 1) {
            count1++;
        }
        else if (cod == 2) {
            count2++;
        }
        else if (cod == 3) {
            count3++;
        }
    }
    printf("MUITO OBRIGADO\n");
    printf("Alcool: %d\n", count1);
    printf("Gasolina: %d\n", count2);
    printf("Diesel: %d\n", count3);

    return 0;
}

File: Course_01/Python/EstruturasRepetitivas/combustivel.py
cod = int(input("Informe um código (1, 2, 3) ou 4 para parar: "))
conte1 = 0
conte2 = 0
conte3 = 0

while cod != 4:
    match cod:
        case 1:
            conte1 += 1
        case 2:
            conte2 += 1
        case 3:
            conte3 += 1
    cod = int(input("Informe um código (1, 2, 3) ou 4 para parar: "))

print("MUITO OBRIGADO")
print(f"Alcool: {conte1}")
print(f"Gasolina: {conte2}")
print(f"Diesel: {conte3}")